src/z3c/recipe/sphinxdoc/__init__.py

```diff
@@ -168,9 +171,11 @@ def install(self):
     update = install
 
 
-def main(projects):
+def main(projects, argv=None, exit_on_error=False):
```

Comment:
I didn't check, but I thought `sphinx.main()` that was there before would exit with an non-zero exit code on error?

As far as I can see, this `main()` is invoked as an entry point, from a new script generated in `bin/`, and I think this script should exit with non-zero if documentation generation fails.